front_end/entrypoints/rn_inspector/rn_inspector.ts

```diff
@@ -13,6 +13,7 @@ import '../../panels/mobile_throttling/mobile_throttling-meta.js';
 import '../../panels/network/network-meta.js';
 import '../../panels/js_profiler/js_profiler-meta.js';
 import '../../panels/rn_welcome/rn_welcome-meta.js';
+import '../../panels/react_devtools/react_devtools-meta.js';
```

Comment:
Looks like this file isn't ready yet and the import statement wasn't intended for this PR. It's crashing the RN Inspector landing page so it's remove it until it exists (or add an empty stub file for now)

Reply:
Oh, you are right, somehow slipped in this PR, will fix that